packages/gafl-webapp-service/src/services/sales-api/sales-api-service.js

```diff
@@ -0,0 +1,35 @@
+import fetch from 'node-fetch'
+import { SALES_API_URL_DEFAULT, SALES_API_TIMEOUT_MS_DEFAULT } from '../../constants.js'
+
+const urlBase = process.env.SALES_API_URL || SALES_API_URL_DEFAULT
+
+const fetchData = async url => {
+  try {
+    const response = await fetch(url.href, {
+      headers: { 'Content-Type': 'application/json' },
+      timeout: process.env.SALES_API_TIMEOUT_MS || SALES_API_TIMEOUT_MS_DEFAULT
+    })
+    return response.json()
+  } catch (err) {
+    // On a failure to connect do not stop the user journey
+    console.error('Unable to connect to the sales API service', err)
+    throw err
+  }
+}
+
+const permitsOperations = {
```

Comment:
These URL's changed recently, remove the `/reference-data` segment from the path.

Also could be simplified to:

```
  fetchPermits: async() => fetchData(new URL('/permits', urlBase)),
  fetchConcessions: async() => fetchData(new URL('/concessions', urlBase)),
  fetchPermitConcessions: async() => fetchData(new URL('/permitConcessions', urlBase))
```

Reply:
I know

Reply:
Will fix